app/views/shared/_dates_and_assignment_header.html.erb

```diff
@@ -1,74 +1,23 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds" id="dates-and-assignment-details">
-    <p class="govuk-body">
-      <strong>Validation date:</strong>
-      <% if @planning_application.valid_from.present? %>
-        <%= time_tag @planning_application.valid_from %>
-      <% else %>
-        Not yet valid
+    <% if @planning_application.consultation.present? && @planning_application.consultation.end_date.present? %>
+      <% if display_or_publish_required?(@planning_application, :site) %>
+        <div id="confirm-site-notice-warning">
+          <%= render(
+                partial: "shared/warning_text",
+                locals: {message: link_to("Confirm site notice display date", edit_planning_application_site_notice_path(@planning_application, @planning_application.site_notice))}
+              ) %>
```

Comment:
How about using the upstream component helper:

```suggestion
          <%= govuk_warning_text(text: link_to("Confirm site notice display date", edit_planning_application_site_notice_path(@planning_application, @planning_application.site_notice))) %>
```

The partial is just the same but more verbose :)

Reply:
I'll get rid of all the others in #1806 :)

Reply:
Thanks for sharing that, I was going to start replacing the others too 😂